src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,145 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> headNode;
+    private Node<T> tailNode;
+    private int currentSize;
```

Comment:
I can't see reason to make more complicated name. We don't have other sizes here so can't really mismatch them
```suggestion
    private int size;
```